src/components/BoosterCard.vue

```diff
@@ -1,4 +1,15 @@
+<script setup>
+    import { defineProps } from "vue"
```

Comment:
Since ya'll are learning a new framework, could you explain to me a little what `defineProps` does?  Is it similar to props in React?  Does it work the same or is anything different?